packages/flutter_tools/test/commands.shard/hermetic/drive_test.dart

```diff
@@ -45,6 +45,38 @@ void main() {
     Cache.enableLocking();
   });
 
+  testUsingContext('warns if screenshot is not supported but continues test', () async {
```

Comment:
FYI @christopherfujino , this is a lot easier than tracking whether each device supports screenshotting in the devicelab codebase

Reply:
SGTM